custom/icds_reports/views.py

```diff
@@ -1653,8 +1627,8 @@ def post(self, request, *args, **kwargs):
         return redirect(self.urlname, domain=self.domain)
 
 
+@location_safe
```

Comment:
This wasn't location_safe before. Why does it need to be now?

Reply:
Intent is to  allow the Bug report view to location restrict users As well. M I missing something here?
@emord

Reply:
So its expected that this is a change in behavior? that was not obvious from this PR

Reply:
Like it was not requested by someone else. I changed with my understanding of Bug report view. Should not it be by default nature.

Reply:
removing this